fvm/evm/stdlib/contract.cdc

```diff
@@ -1 +1,107 @@
-pub contract EVM {}
\ No newline at end of file
+access(all)
+contract EVM {
+
+    /// EVMAddress is an EVM-compatible address
+    access(all)
+    struct EVMAddress {
+
+        /// Bytes of the address
+        access(all)
+        let bytes: [UInt8; 20]
+
+        /// Constructs a new EVM address from the given byte representation.
+        init(bytes: [UInt8; 20]) {
+            self.bytes = bytes
+        }
+    }
+
+    access(all)
+    struct Balance {
+
+        /// The balance in FLOW.
+        access(all)
+        let flow: UFix64
+
+        /// Constructs a new balance, given the balance in FLOW.
+        init(flow: UFix64) {
+            self.flow = flow
+        }
+
+        // TODO:
+        // /// Returns the balance in terms of atto-FLOW.
+        // /// Atto-FLOW is the smallest denomination of FLOW inside EVM.
+        // access(all)
+        // fun toAttoFlow(): UInt64
+    }
+
+    access(all)
+    resource BridgedAccount {
+        access(self)
+        let addressBytes: [UInt8; 20]
+
+        init(addressBytes: [UInt8; 20]) {
+           self.addressBytes = addressBytes
+        }
```

Comment:
does this mean anyone can create a BridgedAccount with the given address? the address has to be allocated by the FVM otherwise another resource can take control of an asset.

Reply:
No, resources can only be created by the contract